scripts/lib/api/processHtml.ts

```diff
@@ -194,12 +194,21 @@ export function convertRubricsToHeaders(
   $: CheerioAPI,
   $main: Cheerio<any>,
 ): void {
```

Comment:
This function would be useful to add unit tests. I only had added them before due to time.